src/main.cpp

```diff
@@ -1071,6 +1072,15 @@ bool ContextualCheckTransaction(
                 }
             }
 
+            if (nu6Active) {
+                for (auto it = fundingStreamElements.begin(); it != fundingStreamElements.end(); ++it) {
+                    if (std::holds_alternative<Consensus::Lockbox>(it->first)) {
+                        // TODO: do we want to do something with the lockbox value here?
+                        fundingStreamElements.erase(it);
```

Comment:
I checked that it is okay to not `break` here (unlike every prior case, where we `break` after erasing the first instance we find, because we're driven by an outer loop over coinbase outputs). Per [the `std::set::erase` documentation](https://en.cppreference.com/w/cpp/container/set/erase):
> References and iterators to the erased elements are invalidated. Other references and iterators are not affected.